### PR TITLE
Add otwrapy

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
cc @felipeam86

File: recipes/otwrapy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+    - numpy
+    - joblib
+    - ipyparallel
+    - openturns
```
Comment (jakirkham) [inline recipes/otwrapy/meta.yaml]: Please cut anything from build that is not strictly necessary at build time.

File: recipes/otwrapy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+    - numpy
+    - joblib
+    - ipyparallel
```
Comment (jakirkham) [inline recipes/otwrapy/meta.yaml]: Should these have the same version constraints that [source]( https://github.com/felipeam86/otwrapy/blob/0.6/setup.py#L25-L26 ) has?

File: recipes/otwrapy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: https://github.com/felipeam86/otwrapy
+  license: LGPLv3
+  license_file: LICENSE
+  summary: General purpose OpenTURNS python wrapper tools
+
+extra:
+  recipe-maintainers:
+    - jschueller
```
Comment (jakirkham) [inline recipes/otwrapy/meta.yaml]: Would you be interested in helping maintain too, @felipeam86 ?

Comment (felipeam86) [inline recipes/otwrapy/meta.yaml]: Sure, I can provide help if needed

Comment (jschueller) [inline recipes/otwrapy/meta.yaml]: ok, added

**Comment by jschueller** [comment]
@jakirkham I made the requested changes

File: recipes/otwrapy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+    - numpy
+    - openturns
```
Comment (jakirkham) [inline recipes/otwrapy/meta.yaml]: Just to confirm, we aren't linking to these? There is no source code we are building. Also we are stuck with having these at build time as it imports `otwrapy` in `setup.py`?

**Comment by jschueller** [comment]
@jakirkham is this good to go ?

**Pull request merged**